fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/leaderboard/LeaderboardIndexTest.java

```diff
@@ -1078,6 +1078,28 @@ public void trimScores() {
         }
     }
 
+    @Test
+    public void mostNegativeHighScoreFirst() {
+        Leaderboards leaderboards = new FlatLeaderboards();
+        basicSetup(leaderboards, true);
+        try (FDBRecordContext context = openContext()) {
+            leaderboards.openRecordStore(context, false);
+
+            assertEquals(4L, leaderboards.evaluateAggregateFunction(leaderboards.timeWindowRankForScore(TimeWindowLeaderboard.ALL_TIME_LEADERBOARD_TYPE, -1), Tuple.from("game-1", Long.MIN_VALUE + 1)).get(0));
+            assertEquals(4L, leaderboards.evaluateAggregateFunction(leaderboards.timeWindowRankForScore(TimeWindowLeaderboard.ALL_TIME_LEADERBOARD_TYPE, -1), Tuple.from("game-1", Long.MIN_VALUE)).get(0));
```

Comment:
Perhaps it might make sense to have one with `BigInteger.valueOf(Long.MIN_VALUE).subtract(BigInteger.ONE))`. Perhaps not, though.

Reply:
Sure, that seems reasonable and seems to work, even though scores aren't normally big ints.